types/params_test.go

```diff
@@ -72,19 +68,22 @@ func makeParams(
 		Validator: ValidatorParams{
 			PubKeyTypes: pubkeyTypes,
 		},
+		Version: VersionParams{ // Needs to be set on creation
```

Comment:
what does that mean?

Reply:
removed. It was there before because proto type was giving me a pointer. Now it's a native go type so its not an issue